server/gfmd/user.c

```diff
@@ -84,6 +91,34 @@ grpassign_add_group(struct group_assignment *ga)
 	u->groups.group_prev = ga;
 }
 
+gfarm_error_t
+user_enter_auth_id(struct user *u, const char *auth_method,
+	const char *auth_user_id, int limit_check,
+	struct user_auth **uap)
+{
+	return (GFARM_ERR_NO_ERROR);
+}
+
+void
+user_set_user_auth_metadata_in_cache(struct user_auth *ua,
+	const char *auth_method, const char *auth_user_id)
+{
+}
+
+struct user_auth
```

Comment:
remove the user_lookup_auth_id() function,
we don't have to export struct user_auth.